tf2/include/tf2/exceptions.h

```diff
@@ -33,9 +33,21 @@
 #define TF2_EXCEPTIONS_H
 
 #include <stdexcept>
+#include <cstdint>
 
 namespace tf2{
 
+enum class TF2Error : std::uint8_t{
```

Comment:
nitpick: `enum class TF2Error : std::uint8_t { ` use a whitespace before the `{`